run.sh

```diff
@@ -0,0 +1,57 @@
+# Argument Parsing
+echo "Positional Parameters"
+echo 'Github Project Repository: ' $1
+echo 'Github Project Repository Module Path: ' $2
+
+if [ "$1" == "" ]; then
+    echo "arg1 - github project repository"
+    exit
+elif [ "$2" == "" ]; then
+    echo "arg2 - github project module path"
+    exit
+fi
+
+# Setup Variables
+crnt=`pwd`
+working_dir=`dirname $0`
+project_repository=$1
+project_module_path=${crnt}/$2
+
+# TestRunner
+if [ ! -d "testrunner" ]; then
+    git clone https://github.com/ReedOei/testrunner.git
+fi
+cd testrunner
+mvn install
+cd ..
+
+# Dt-Fixing-Tools
+if [ ! -d "dt-fixing-tools" ]; then
+    git clone https://github.com/ReedOei/dt-fixing-tools.git
+fi
+cd dt-fixing-tools
+mvn install
+cd ..
+
+# Project Repository
+if [ ! -d "$project_module_path" ]; then
+    git clone ${project_repository}.git
+fi
+cd ${project_module_path}
+{ time -p mvn test -fn |& tee mvn-test.log ;} 2> mvn-test-time.log
+cd ${crnt}
+
+# Setup Necessary Scripts
+if [ ! -f "modify-project.sh" ]; then
```

Comment:
Similar to above, the user must have already cloned the repository to have this script, so will already have `modify-project.sh` and `PomFile.java`.

Reply:
Ok! I'll navigate over to those scripts